### PR TITLE
Refactor how the status icons are rendered

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Completely refactored how the status icons are rendered. In some ways this is less elegant; it's obviously much nicer to be able to drop in an existing glyph from Font Awesome and color it vs. having to draw circle backgrounds with CSS and tweak the absolute position of the glyphs so they are centered. But this will be more flexible moving forward as we won't be limited to only the Font Awesome icons that have baked-in circle backgrounds.

On the plus side, using flexbox to lay out this little section of the alert is "better" than adding `::before` content to the `h3` and pulling it left with a negative margin.

## Testing done
Local

## Screenshots
Previews of the five possible status types (icons and color combos) are shown below
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/dashboard/sass/dashboard-alert.scss [hunk 1]
```diff
@@ -31,24 +54,35 @@
 
   section.status {
     margin: 1.5em 0;
-    padding: 1em 0 1em 2em;
+    padding: 1em 0;
```
Comment (jaredcunha-usds) [inline src/applications/personalization/dashboard/sass/dashboard-alert.scss]: We're trying to start using consistent rhythms in horizontal/vertical spacing. Any chance you'd be able to use the spacing tokens instead? https://design.va.gov/design/spacing-units

These are really useful on margins and padding, but can also be set to width and height in some instances as well.

Comment (erikphansen) [reply src/applications/personalization/dashboard/sass/dashboard-alert.scss]: I just pushed up a change that addresses this.

**Comment by Samara-Strauss** [comment]
Screenshots look 😍! Great work!

File: src/applications/personalization/dashboard/sass/dashboard-alert.scss [hunk 2]
```diff
@@ -57,61 +90,49 @@
...
+  );
+
+  @include make-status-styles(
+    $status: "enrolled",
+    $color: $color-green,
+    $icon: $fa-var-check,
+    $top: 1px,
+    $left: 2.5px,
+    $size: 14px,
+  );
```
Comment (ncksllvn) [inline src/applications/personalization/dashboard/sass/dashboard-alert.scss]: Ah very cool mixin!

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: src/applications/personalization/dashboard/sass/dashboard-alert.scss (hunk 2 above)
Comment (erikphansen) [reply src/applications/personalization/dashboard/sass/dashboard-alert.scss]: more or less stole it from the existing `make-hub` mixin :)

**Pull request merged**